--- thunar-volman-0.1.2.orig/thunar-volman/tvm-block-device.c	2007-01-20 11:57:16.000000000 +0100
+++ thunar-volman-0.1.2/thunar-volman/tvm-block-device.c	2007-07-31 08:03:07.664805395 +0200
@@ -88,6 +88,7 @@
 {
   gboolean result = FALSE;
   gboolean autoipod;
+  gboolean is_audio_player = FALSE;
   gchar   *autoipod_command;
   gchar   *autophoto_command;
   gchar   *storage_udi;
@@ -106,6 +107,7 @@
           /* check if we have a portable audio player here */
           if (libhal_device_query_capability (context, storage_udi, "portable_audio_player", NULL))
             {
+	      is_audio_player = TRUE;
               /* check if we have an iPod here */
               product = libhal_device_get_property_string (context, storage_udi, "info.product", NULL);
               if (product != NULL && strcmp (product, "iPod") != 0)
@@ -151,7 +153,8 @@
           else
             {
               /* no photos, so we can manage only music */
-              response = TVM_RESPONSE_MUSIC;
+              if(is_audio_player)
+		      response = TVM_RESPONSE_MUSIC;
             }
 
           /* check what to do */
